### PR TITLE
Lesson 14

### PR DESCRIPTION
**Pull request by namelexa**
### Fixed Issues (if relevant)

1. magento/magento2#<issue_number>: Issue title
2. ...

### Manual testing scenarios

1. ...
2. ...

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: app/code/AKisilenko/ModuleLesson6/Console/Command/ProductQuantityUpdate.php [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+     * @throws NoSuchEntityException
+     */
+    public function updateProductStock(
+        $productId,
+        $stockData
+    ) {
+        $product = $this->productRepository->getById($productId);
+        $stockItem = $this->_stockRegistry->getStockItem($product->getId());
+        $stockItem->setData('is_in_stock', 1);
+        $stockItem->setData('quantity', $stockData);
```
Comment (chabanenk0) [inline app/code/AKisilenko/ModuleLesson6/Console/Command/ProductQuantityUpdate.php]: Incorrect field name "quantity" Please rename to 'qty'. Please check fields of the  table 'cataloginventory_stock_item'; 
cataloginventory_stock_item

File: app/code/AKisilenko/ModuleLesson6/Console/Command/ProductQuantityUpdate.php [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        $stockItem->setData('manage_stock', 1);
+        $stockItem->setData('notify_quantity', 1);
```
Comment (chabanenk0) [inline app/code/AKisilenko/ModuleLesson6/Console/Command/ProductQuantityUpdate.php]: Incorrect field name 'notify_quantity', did you mean 'notify_stock_qty'?

Comment (namelexa) [reply app/code/AKisilenko/ModuleLesson6/Console/Command/ProductQuantityUpdate.php]: No, I mean 'use_config_notify_stock_qty'. Already fixed it

File: app/code/AKisilenko/ModuleLesson6/Console/Command/ProductQuantityUpdate.php (hunk 1 above)
Comment (namelexa) [reply app/code/AKisilenko/ModuleLesson6/Console/Command/ProductQuantityUpdate.php]: All done

**Pull request merged**